Justfile

```diff
@@ -0,0 +1,16 @@
+flox_bats_tests_invocation := "flox run '.#flox-tests' -- -- --flox target/debug/flox"
+
+_default:
+    @just --list --unsorted
+
+# Run the 'bats' test suite
+shell-test +test_files="":
+    @cargo build
+    @{{flox_bats_tests_invocation}} {{test_files}}
+
+# Run the Rust unit tests
+unit-test +regex="":
+    @cargo test --workspace {{regex}}
```

Comment:
🤔 should this enable extra-tests?

Reply:
I'd rather make a separate command called `impure-tests` or something like that just so that we can easily do both